app/locales/en/translations.js

```diff
@@ -0,0 +1,233 @@
+export default {
```

Comment:
Incidentally, the `ember-osf` addon (which provides some core features) has some translations of its own. Although the addon has too few translations to be noticeable yet, the goal was that the addon and consuming app would seamlessly combine their translations. 

https://github.com/CenterForOpenScience/ember-osf/tree/develop/addon/locales/en-us

In the addon, we use `en-us` for the locale name. It might be useful to pick a single consistent locale name going forward, so that all parts of the app can respect a single locale setting.